### DISCUSSION TITLE
Help of window 11

### DISCUSSION THREAD

**Discussion by bikram101** [pyscada/PyScada#139] [General]
Hi, I am new to python  and django still learning. However, I want to make window web app to interact with bunch of modbus devices kind of HMI. I installed pip pyscada, django framework and also nginx webserver on my window 11. Is it this library work with windows and how I can open pyscada on my window11?

**Comment by clavay** [comment]
For Windows as the OS don't implement the [os.fork](https://docs.python.org/3/library/os.html#os.fork) you should use the docker installation.

Please use this [folder](https://github.com/clavay/PyScada/tree/dockerSQLiteWindows/dockerSQLite), look at the readme file.

**Comment by clavay** [comment]
Reopen this discussion if needed.
